README.md

```diff
@@ -1,2 +1,18 @@
-# sample-julia-genie
-Julia with Genie Devcontainer Sample
+# Sample Julia App with Genie.jl
+
+This is a sample Julia application using the Genie.jl framework, designed to run in a dev container.
+
+## Setup Instructions
```

Comment:
Please update the setup instructions to include how to install Daytona and run this with `daytona create REPO_URL`.